### PR TITLE
[TECH] Modal-window

### PR DESCRIPTION
**Pull request by kseniyaLine**

### CODE REVIEW

**Review by tensenrey — DISMISSED** [review]

**Review by tensenrey — DISMISSED** [review]

File: src/components/modal-w/Modal.scss [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  min-width: 440px;
+  min-height: 126px;
+  background-color: white;
+}
+
+.modalMain {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
```
Comment (mtalstykh-exadel) [inline src/components/modal-w/Modal.scss]: move styles to the src/styles/modal.scss

File: src/components/buttons/Buttons.jsx [hunk 2]
```diff
@@ -1,10 +1,11 @@
...
-import { makeStyles } from "@material-ui/core";
-import { Link } from "react-router-dom";
-import { Trans } from '@lingui/macro';
-import { switchLang } from "../../utils/lang-service.js";
-
+import {makeStyles} from "@material-ui/core";
+import {Link} from "react-router-dom";
+import {Trans} from '@lingui/macro';
+import {switchLang} from "../../utils/lang-service.js";
+import SimpleModal from "../modal-w/Modal.js";
```
Comment (mtalstykh-exadel) [inline src/components/buttons/Buttons.jsx]: use native material modal

File: src/components/layout/Layout.js [hunk 3]
```diff
@@ -4,11 +4,11 @@ import React from "react";
 import { PropTypes } from "prop-types";
 import Header from '../../components/header/header';
 
-const Layout = ({children}) => {
+const Layout = ({pageWrapperClass},{children}) => {
```
Comment (mtalstykh-exadel) [inline src/components/layout/Layout.js]: you can transform it into { pageWrapperClass, children }

File: src/components/layout/Layout.js [hunk 4]
```diff
@@ -4,11 +4,11 @@ import React from "react";
...
   return (
     <>
       <Header/>
-      <main className="main">
+       <main className={pageWrapperClass}>
```
Comment (mtalstykh-exadel) [inline src/components/layout/Layout.js]: remove extra space

File: src/components/buttons/Buttons.jsx [hunk 5]
```diff
@@ -1,12 +1,13 @@
...
-import { makeStyles } from "@material-ui/core";
-import { Link } from "react-router-dom";
-import { Trans } from '@lingui/macro';
-import { switchLang } from "../../utils/lang-service.js";
+import {makeStyles, Modal} from "@material-ui/core";
+import {Link} from "react-router-dom";
+import {Trans} from '@lingui/macro';
+import {switchLang} from "../../utils/lang-service.js";
+import PropTypes from 'prop-types';
+import '../../styles/modal.scss';
```
Comment (mtalstykh-exadel) [inline src/components/buttons/Buttons.jsx]: import it not here but in App.js

File: src/components/buttons/Buttons.jsx [hunk 6]
```diff
@@ -20,25 +21,50 @@ const Buttons = () => {
...
+  const handleOpen = () => {
+    setOpen(true);
+  };
+
+  const handleClose = () => {
+    setOpen(false);
+  };
   return (
     <>
+      <div id='modal'/>
```
Comment (mtalstykh-exadel) [inline src/components/buttons/Buttons.jsx]: why do you need the self-closing div?)

**Review by mtalstykh-exadel — DISMISSED** [review]

**Review by tensenrey — DISMISSED** [review]

**Review by tensenrey — DISMISSED** [review]

**Review by mtalstykh-exadel — APPROVED** [review]

**Review by tensenrey — APPROVED** [review]

**Pull request merged**